cfc-testing/run-all-benchmarks.sh

```
@@ -0,0 +1,218 @@
+run_stress_test() {
+    echo -e "${GREEN}🚀 Running Stress Test Benchmark${NC}"
+    echo -e "${GREEN}===============================${NC}"
+    
+    echo -e "${YELLOW}📊 Running rapid synthesis stress test${NC}"
+    echo ""
+    
+    for i in {1..5}; do
+        echo -e "${YELLOW}Stress Test Iteration $i/5${NC}"
+        rm -f deployment-context.json
+        
+        local start_time=$(date +%s.%N)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)